requirements-other.txt

```diff
@@ -0,0 +1,2 @@
+pickle5
```

Comment:
This should really be in the main requirements, shouldn't it?

Reply:
I just picked the file in the version of the code before yesterday, and replicated that field as it was.

Reply:
But I would say that yes, it has to be in the main requirements because it is used in the pre-processing. Do you want me to change that here or in a separate PR?

Reply:
Let's do it here